build/Dockerfile

```diff
@@ -15,7 +16,9 @@ RUN mvn clean install
 
 # our final base image
 #FROM eclipse-temurin:18.0.1_10-jre
-FROM eclipse-temurin:19.0.1_10-jre
+#FROM eclipse-temurin:19.0.1_10-jre
+
+FROM eclipse-temurin:latest
```

Comment:
:latest here is probably not a good idea.

Reply:
Image tag name change to eclipse-temurin:19-jdk-jammy

Reply:
Image tag name change to eclipse-temurin:19-jdk-jammy